lib/ansible/modules/web_infrastructure/ansible_tower/tower_group.py

```diff
@@ -110,66 +90,76 @@ def main():
         description=dict(),
         inventory=dict(required=True),
         variables=dict(),
-        credential=dict(),
-        source=dict(choices=["manual", "file", "ec2", "rax", "vmware",
-                             "gce", "azure", "azure_rm", "openstack",
-                             "satellite6", "cloudforms", "custom"], default="manual"),
-        source_regions=dict(),
-        source_vars=dict(),
-        instance_filters=dict(),
-        group_by=dict(),
-        source_script=dict(),
-        overwrite=dict(type='bool', default=False),
-        overwrite_vars=dict(),
-        update_on_launch=dict(type='bool', default=False),
```

Comment:
These did move to the inventory source model, but people using older versions of Tower still need to use these parameters on the group model. Because of that, I don't know if we can just get rid of them. It may require a deprecation schedule.

Reply:
Ah that makes sense. Because the parameters were just passed straight through to the tower_cli function with **kwargs I assumed they had never done anything. Didn't think of previous versions. I will revert that commit so they can be deprecated in the future.

Reply:
@AlanCoding Whats the process for starting to deprecate these. Or should we just leave them be as they don't break anything

Reply:
Maybe we could just remove them, if it was put in the release notes?